apps/frontend/core-dapp/src/components/PpoAmount.tsx

```diff
@@ -24,18 +24,26 @@ const PpoAmount: React.FC<{
   iconSpacing?: number
   signDisplay?: 'exceptZero' | 'never'
 }> = ({ amount, iconSize = '12', iconSpacing = 2, signDisplay = 'exceptZero' }) => {
-  const parts = Intl.NumberFormat(undefined, {
-    maximumFractionDigits: 2,
-    minimumFractionDigits: 1,
-    notation: 'compact',
-    signDisplay,
-  }).formatToParts(+(amount ?? '0'))
-
-  const signCharacter = parts.find(isSign)?.value
-  const amountCharacters = parts
-    .filter(isNotSign)
-    .map(({ value }) => value)
-    .join('')
+  let signCharacter
+  let amountCharacters
```

Comment:
could we type these or assign a default value?
```suggestion
  let signCharacter = ''
  let amountCharacters = ''
```

Reply:
I don't see a reason to do that.

* Typescript is smart and detects that in all `if/else` cases the value is being initialized, so it doesn't add `undefined` to the type.
* Likewise, Typescript can infer that the type of those values will be `string` based on what's being assigned to them.

It would be equivalent to doing:

```ts
const signCharacter = typeof amount === 'string' && +amount < 0.01 && +amount > 0
  ? (signDisplay === 'exceptZero' ? '+' : '')
  : parts.find(isSign)?.value
```